kr/simulation/jmavsim.md

```diff
@@ -0,0 +1,88 @@
+# jMAVSim with SITL
```

Comment:
Not translated. This page was previously named sitl.md, but has been updated "a little"